lib/ansible/modules/network/eos/eos_acls.py

```diff
@@ -0,0 +1,840 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for eos_acls
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = """
+---
+module: eos_acls
+version_added: '2.10'
+short_description: 'Manages IP access-list attributes of Arista EOS interfaces'
+description: This module manages the IP access-list attributes of Arista EOS interfaces.
+author: Gomathiselvi S (@GomathiselviS)
+notes:
+- Tested against Arista vEOS v4.20.10M
+options:
+  config:
+    description: A dictionary of IP access-list options
+    type: list
+    elements: dict
+    suboptions:
+      afi:
+        description:
+          - The Address Family Indicator (AFI) for the Access Control Lists (ACL).
+        type: str
+        required: true
+        choices: ['ipv4', 'ipv6']
+      acls:
+        description:
+        - A list of Access Control Lists (ACL).
+        type: list
+        elements: dict
+        suboptions:
+          standard:
+            description: standard access-list or not
+            type: bool
+            default: False
+          name:
+            description: Name of the acl-list
+            type: str
+            required: true
+          aces:
+            description: Filtering data
+            type: list
+            elements: dict
+            suboptions:
+              sequence:
+                description: sequence number for the ordered list of rules
+                type: int
+              remark:
+                description: Specify a comment
+                type: str
+              fragment_rules:
+                description: Add fragment rules
+                type: bool
+              grant:
+                description: Action to be applied on the rule
+                type: str
+                choices: ['permit', 'deny']
+              line:
```

Comment:
Wondering if https://github.com/ansible/ansible/pull/66207/files#diff-fa14da51ccc1047e3647da9c0179586fR87 sounds good? Also, I added `ace` as an alias to line. Is that something we want to do?

Reply:
Added alias